_posts/2019-01-14-graphql-union-vs-interface.md

```diff
@@ -16,15 +16,6 @@ with GraphQL that can make the consuming our data lot easier.
 
 ## GraphQL: Interface or Union?
 
-At Artsy we’ve been moving towards GraphQL for all of our new services. Acknowledging that GraphLQ is a relatively
-new technology, we faced some challenging questions as we were developing one our most recent services.
-
-<!-- more -->
-
-Naively with my first attempt to define GraphQL types and schemas, I naturally tried to map our database models to
-GraphQL types. While this may work for lot of cases, we may not be utilizing some of the most useful features that
-comes with GraphQL.
-
```

Comment:
This makes sense, only thing is, can you move `<!-- more -->` to after first paragraph? maybe https://github.com/artsy/artsy.github.io/pull/523/files#diff-20a0df124e0c7c000bd0f9e376dec9c0R20 ?

Reply:
My bad was not aware of the usage of the comment. I updated it.